src/context/AuthContext.tsx

```diff
@@ -6,18 +6,20 @@ export const AuthContext = createContext(null);
 export default function AuthContextProvider(props: any) {
   const [loginData, setLoginData] = useState(null);
   const baseUrl = "https://upskilling-egypt.com:3003/api/v1";
+
   const saveLoginData = () => {
```

Comment:
We remove the decoding portion and replacing it by an API call to the getCurrentUser api. This API will contain all information needed for the user including the user's avatar that will be used in the navbar.